### DISCUSSION TITLE
config for nodemcuv3

### DISCUSSION THREAD

**Discussion by Poltavtcev** [Dilbert66/esphome-dsckeybus#42] [Q&A]
Hello everyone. Thank you very much for the great work that has been done! I'm very excited and can't wait for it to work for me)
I am using Nodemsu3 and when I try to flash it, I get the following error: 

```
Failed config

status_led: [source /config/esphome/DscAlarm.yaml:36]
  pin:  [source /config/esphome/DscAlarm.yaml:37]
    
    Cannot resolve pin name 'D4' for board nodemcuv3.
    number: D4 [source /config/esphome/DscAlarm.yaml:37]
    inverted: True
```

Please help me edit the config file for this board correctly. I'm just starting an adventure in esp.
Many thanks!

**Comment by Dilbert66** [comment]
use "GPIO2" instead of D4 for the pin name:

pin: GPIO2

**Reply by Poltavtcev** [reply]
"Error fetching configuration information": https://clx.by/s/OXhGm 
Thanks, but it looks like I have a long way to go (
What would you advise to read or who to turn to for help to overcome these mistakes?
